hw/ip/prim/prim_flash.core

```diff
@@ -10,6 +10,7 @@ filesets:
     depend:
       - lowrisc:prim:prim_pkg
       - lowrisc:prim:primgen
+      - lowrisc:prim:ram_1p
```

Comment:
Ah, now I see where the dependency comes from: prim_generic_flash (the non-generic flash macro won't need the RAM primitive hopefully).

Can you add the dependency into `hw/ip/prim_generic/prim_generic_flash.core`?

(It would be that our dynamic primitives system messes things up and the dependency doesn't work there, but it should; otherwise it's a bug I'd like to know about.)

Reply:
Done.